Libs/MRML/Widgets/Resources/UI/qMRMLVolumeThresholdWidget.ui

```diff
@@ -49,17 +49,31 @@
      </item>
     </widget>
    </item>
-   <item row="1" column="0" colspan="2">
+   <item row="1" column="0" colspan="3">
     <widget class="ctkRangeWidget" name="VolumeThresholdRangeWidget"/>
    </item>
+   <item row="1" column="3">
+    <widget class="QToolButton" name="RangeButton">

```

Comment:
Please use the same button type and size constraints for the threshold `...` as for the W/L `...` (so that the `...` buttons appear the same size). See how they appear on my computer:

![image](https://user-images.githubusercontent.com/307929/120908935-7600ca00-c63d-11eb-8374-8c93d87b39f9.png)

Reply:
I have pushed an update to make these look the same. See example image below:
![image](https://user-images.githubusercontent.com/15837524/120936595-9761c480-c6d6-11eb-8dee-767e16df92a1.png)